bgpd/bgp_evpn.c

```diff
@@ -6144,6 +6144,12 @@ static bool bgp_evpn_remote_ip_hash_cmp(const void *p1, const void *p2)
 	const struct evpn_remote_ip *ip1 = p1;
 	const struct evpn_remote_ip *ip2 = p2;
 
+	if (ip1 == NULL && ip2 == NULL)
```

Comment:
No need to compare NULL values for hash functions.

Reply:
Fixed.